inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp

```diff
@@ -640,6 +694,568 @@ struct jit_uni_interpolate_kernel_f32 : public jit_uni_interpolate_kernel, publi
         L(tail_loop_end_label);
     }
 
+    void cubic_c_gathered() {
```

Comment:
This function contains a lot of code duplication. Could you please avoid it? The same for `cubic_planar`

Reply:
fixed. Thank you.